entrypoint.sh

```diff
@@ -4,10 +4,13 @@ cd "${GITHUB_WORKSPACE}" || exit
 
 export REVIEWDOG_GITHUB_API_TOKEN="${INPUT_GITHUB_TOKEN}"
 
-if [ "${INPUT_REPORTER}" = x"github-pr-review" ]; then
-  markdownlint "${INPUT_MARKDOWNLINT_FLAGS:-.}" |
-    reviewdog -efm="%s:%l:%c %m" -name="markdownlint" -diff="git diff HEAD^" -reporter=github-pr-review -level="${INPUT_LEVEL}" -tee
-else
-  markdownlint "${INPUT_MARKDOWNLINT_FLAGS:-.}" |
-    reviewdog -efm="%f:%l:%c %m" -name="markdownlint" -diff="git diff HEAD^" -reporter=github-pr-check -level="${INPUT_LEVEL}" -tee
-fi
+markdownlint "${INPUT_MARKDOWNLINT_FLAGS:-.}" |
+  reviewdog \
```

Comment:
shouldn't this line also end with `/`?

```suggestion
markdownlint "${INPUT_MARKDOWNLINT_FLAGS:-.}" \
  | reviewdog \
```

Reply:
Oh that's good catch. I wonder why it worked as-is.
Probably the bash feature? I'll fix it anyway.